Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -96,35 +97,40 @@ bool AVIDump::CreateVideoFile()
 {
   const std::string& s_format = g_Config.sDumpFormat;
 
-  std::stringstream file_ss;
-  file_ss << File::GetUserPath(D_DUMPFRAMES_IDX)
-                   << "framedump" << s_file_index
-                   << "." << s_format;
-  std::string filename = file_ss.str();
-  File::CreateFullPath(filename);
+  std::string s_dump_path = g_Config.sDumpPath;
```

Comment:
We're going to rely on the const-reference-lifetime-extension thing? Because later we might assign a temporary to it.